package.json

```diff
@@ -82,6 +84,21 @@
                 },
                 "editor.autoIndent": false
             }
+        },
+        "commands": [
+            {
+                "command": "configure-pipeline",
+                "title": "Configure Pipeline",
+                "category": "Azure Pipelines"
```

Comment:
should the category name be Azure DevOps?

Reply:
Its more related to Azure Pipelines hence kept the same.